src/aem/azext_aem/__init__.py

```diff
@@ -0,0 +1,48 @@
+# --------------------------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for license information.
+# --------------------------------------------------------------------------------------------
+
+from azure.cli.core import AzCommandsLoader
+
+import azext_aem._help  # pylint: disable=unused-import
+
+
+class AEMCommandsLoader(AzCommandsLoader):
+
+    def __init__(self, cli_ctx=None):
+        from azure.cli.core.commands import CliCommandType
+        aem_custom = CliCommandType(
+            operations_tmpl='azext_aem.custom#{}')
+        super(AEMCommandsLoader, self).__init__(cli_ctx=cli_ctx,
+                                                custom_command_type=aem_custom)
```

Comment:
Set min profile here like with other command modules so it doesn't appear on the stack profile.

Reply:
Thanks. I put a comment in the code to do this, but looks like i still forgot it